plugins/redhat.vscode-openshift-connector/0.0.17/meta.yaml

```diff
@@ -1,14 +1,17 @@
-id: redhat.vscode-openshift-connector
+publisher: ""
+name: redhat.vscode-openshift-connector
 version: 0.0.17
 type: VS Code extension
-name: OpenShift Connector
+displayName: OpenShift Connector
 title: OpenShift Connector
 description: Interacting with Red Hat OpenShift clusters and providing a streamlined developer experience using Eclipse Che
 url: https://marketplace.visualstudio.com/_apis/public/gallery/publishers/redhat/vsextensions/vscode-openshift-connector/0.0.17/vspackage
 icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
-publisher: Red Hat, Inc.
 repository: https://github.com/redhat-developer/vscode-openshift-tools
 category: Other
 firstPublicationDate: "2019-03-11"
 attributes:
   containerImage: "eclipse/che-remote-plugin-openshift-connector-0.0.17:next"
+deprecate:
+  autoMigrate: false
+  migrateTo: redhat/vscode-openshift-connector/0.0.17
```

Comment:
You have another plugin with the repository from `redhat-developer` organization and publisher there is `redhat-developer` https://github.com/eclipse/che-plugin-registry/pull/111/files#diff-eee18f0b9794223fd4f75a2007aeed2fR1
Here you use `publisher: redhat`.
Should publishers be the same for these plugins? If no - could you please elaborate more, when `redhat` and `redhat-developer` values should be used for publisher field?

Reply:
I believe it changed URL of archive from VS Code marketplace to github. I didn't change publisher after that. Actually, I don't know what should we do in this case.